<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RSAbtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AESbtn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SHA256btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA256</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>531</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="RSAPage">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>451</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="RSAKeyBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Générer une paire de clés</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RSAencryptBtn">
         <property name="text">
          <string>Chiffrer un fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RSAdecryptBtn">
         <property name="text">
          <string>Déchiffrer un fichier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="AESPage">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>451</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="AESKeyBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Générer une clé secrète</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AESencryptBtn">
         <property name="text">
          <string>Chiffrer un fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AESdecryptBtn">
         <property name="text">
          <string>Déchiffrer un fichier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="ShaPage">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>451</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="ShaCalcBtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Obtenir le hash (sha256) d'une chaine de caractère (dans un fichier)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ShaCalcFileBtn">
         <property name="text">
          <string>Obtenir le hash Sha256 d'un fichier</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="changeBaseDirButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Changer de répertoire</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
